client/src/i18n/fr/patient_records.json

```diff
@@ -1,31 +1,51 @@
-"RECENT":"Dernières Visites",
-"IN_PROGRESS":"En cours",
-"SEARCH_INFO":"L'entrée recherche le diagnostic après avoir tapé 3 caractères."},
-"FINANCIAL_ACTIVITY":{"ACCOUNT":"Numéro de compte",
-"DEBTOR_GROUP":"Groupe débiteur",
-"REPORT_TITLE":"Activité financière du Patient"},
-"STAT_TOTAL_PATIENT":"Total Patients",
-"STAT_TOTAL_VISIT":"Total Visits",
-"STAT_TOTAL_NEW":"Nouveaux Patients",
-"IS_COVERED_BY":"est couvert par",
-"INVOICE_COVERED_BY":"Cette facture est couverte par",
-"TITLE":"Patients",
-"UPDATE_PICTURE":"Mettre à jour la Photo",
-"UPLOAD_PICTURE":"Télécharger Photo Patient",
-"NOT_FOUND":"Impossible de trouver le patient ayant ce ID. Veuillez contacter votre administrateur systeme",
-"DOB_NOT_SPECIFIED":"Seulement une année de naissance a été spécifiée lors de l'enregistrement du patient. C'est une date approximative."}}
\ No newline at end of file
+{
+  "PATIENT_RECORDS":{
+    "CHECK_IN":{
+      "TITLE":"Visites",
+      "SUBMIT":"Confirmer la visite du patient",
+      "NOT_RECENT":"Ce patient n'a pas été à l'hopital récemment, vérifier le patient pour le suivi de cette visite à l'hôpital.",
+      "ORIGINAL":"Ce patient n'a jamais été consulté à l'hopital.",
+      "RECENT":"Visites récentes",
+      "REPORT_TITLE":"Rapport de consultations des patients",
+      "ITEM":"Confirmée",
+      "BY":"par",
+      "SUCCESS":"Confirmation de la visite du patient avec succes"
+    },
+    "VISITS":{
+      "TITLE":"Visites des patients",
+      "ADMIT":"Admission à l'hôpital",
+      "ADMISSION_DIAGNOSIS":"Diagnostic Admission",
+      "ADMISSION":"Admission",
+      "DISCHARGE":"Décharge",
+      "ORIGINAL":"Ce patient n'a pas de visites enregistrées",
+      "RECENT":"Dernières Visites",
+      "IN_PROGRESS":"En cours",
+      "SEARCH_INFO":"L'entrée recherche le diagnostic après avoir tapé 3 caractères."
+    },
+    "FINANCIAL_ACTIVITY":{
+      "ACCOUNT":"Numéro de compte",
+      "DEBTOR_GROUP":"Groupe débiteur",
+      "REPORT_TITLE":"Activité Financière du Patient"
+    },
+    "STAT_TOTAL_PATIENT":"Total Patients",
+    "STAT_TOTAL_VISIT":"Total Visits",
+    "STAT_TOTAL_NEW":"Nouveaux Patients",
+    "IS_COVERED_BY":"est couvert par",
+    "INVOICE_COVERED_BY":"Cette facture est couverte par",
+    "TITLE":"Patients",
+    "UPDATE_PICTURE":"Mettre à jour la Photo",
+    "UPLOAD_PICTURE":"Télécharger Photo Patient",
+    "NOT_FOUND":"Impossible de trouver le patient ayant ce ID. Veuillez contacter votre administrateur systeme",
+    "DOB_NOT_SPECIFIED":"Seulement une année de naissance a été spécifiée lors de l'enregistrement du patient. C'est une date approximative.",
+    "FINANCES" : {
+      "IN_GOOD_STANDING" : "Tout a l'air normal!",
+      "NOT_IN_GOOD_STANDING" : "Il y a quelques irrégularités dans l'activité financière.",
+      "HAS_CREDITOR_BALANCE" : "Le patient a un solde créditeur (négatif) de <b>({{ balance | currency:currency_id}})</b>. Cela peut être intentionnel, mais peut indiquer un problème.",
```

Comment:
Instead of having `créditeur (négatif)` we can just have `créditeur` or `négatif`; because if we read just `créditeur négatif` it is a bit confusing, and we know that `solde créditeur` =  `solde negatif`.

Example : 
`Le patient a un solde créditeur de <b>({{ balance | currency:currency_id}})</b>. Cela peut être intentionnel, mais peut aussi indiquer un problème.`